deliver/lib/deliver/setup.rb

```diff
@@ -31,7 +31,7 @@ def setup_deliver(file_path, data, deliver_path, options)
 
         # Add a README to the screenshots folder
         FileUtils.mkdir_p(screenshots_path) # just in case the fetching didn't work
-        File.write(File.join(screenshots_path, 'README.txt'), File.read("#{Deliver::ROOT}/lib/assets/ScreenshotsHelp"))
+        File.write(File.join(screenshots_path, 'README.md'), File.read("#{Deliver::ROOT}/lib/assets/ScreenshotsHelp"))
```

Comment:
I'm unsure about making this change since it will add an additional file in everyone's repos who already have a `README.txt` in there 🤔 I'd either say revert or only create an `README.md` if no `README.txt` 🤷‍♂️

Reply:
Oh wow sorry for the late reply! I'm still catching up with GitHub notifications 🙈 

Good point about existing files, possibly involving gitignored files etc 😅. If we keep the .txt extension, should we revert all the markdown formatting that I added? 

I think I'd rather go via the 2nd option you suggested, but the same markdown formatting question stands 😄 Thoughts? No biggie to leave the markdown formatting?

Reply:
I can also revert the extension and all the formatting if this is gonna be too much work 👀 I was thinking it just reads slightly better with formatting, but ¯\_(ツ)_/¯

Reply:
The formatting is totally fine for a txt file in my opinion 😊 I don't render most markdown files anyway. I just few them like this 🤷‍♂️

Reply:
SGTM! I'll revert the extension only then :)

Reply:
Done 😊 Lemme know if there's anything else blocking this PR @joshdholtz 🙌